src/pages/index.js

```diff
@@ -419,20 +408,21 @@ const handleArrowClick = (e) => {
   // get the id of the element to scroll to. e.g. "#aboutsection"
   const targetQuery = e.currentTarget.getAttribute('href');
 
-  // scoll the element into view
+  // scroll the element into view
   document.querySelector(targetQuery).scrollIntoView({ behavior: 'smooth' });
 
   // set the URL hash to the href attribute
-  if (history.pushState) {
-    history.pushState(null, null, targetQuery);
+  if (window.history.state) {
```

Comment:
This disables smooth scrolling. Please change it back or make sure that it's working.

Reply:
@saranshbarua But I didn't make any changes to index.js .  Is it due to any module?

Reply:
Wait, what? I thought you refactored the code.

Reply:
No, just solved the issue #45 to deploy it to netlify.

Reply:
Maybe surely can take up refactoring in further PRs.

Reply:
Did you run a yarn upgrade?

Reply:
No. Just npm install.

Reply:
Ok, these changes were made by @Yugandhartripathi in a [PR](https://github.com/osdc/osdc.github.io/pull/44/files) maybe he could share some insights.

Reply:
And we decided to use yarn instead of npm in https://github.com/osdc/osdc.github.io/pull/44#pullrequestreview-195264932. So we don't need package-lock.json

Reply:
@sidntrivedi012 just change state to pushState here and it should work. I don't remember why I did this at the time but well we can fix it now.

Reply:
@Yugandhartripathi Changed `state` to `pushState`. Yet no luck.

Reply:
Please check it now. I guess it has started working now.

Reply:
LGTM :+1:

Reply:
Yes, it works fine for me. Nice work.